### DISCUSSION TITLE
Crucial World Exporting Feature

### DISCUSSION THREAD

**Discussion by ClockTheSenior** [Sam54123/mc-world-export#45] [Ideas]
Hello! It is really nice that we can select the chunk radius! But do you know what would be better? Having a World Exporting Feature like Mineways or Mine-Imator! Honestly this is really a very important feature because this can highly optimize the exporting-importing time. I know this is not a simple thing to implement, but this is a very crucial feature. I would suggest looking at Mine-Imator's World Importer or Mineways' World Exporter. / /
![image](https://user-images.githubusercontent.com/106481085/224487943-c5defaff-ba45-4751-ba81-aae5c5aca1c7.png)

**Comment by Igrium** [comment]
That's currently planned for the next update! Right now, I can't decide between a 3D bounding box selection or a 2D area selection.

**Reply by ClockTheSenior** [reply]
honestly, 3d would be simpler to implement since you could add a thing like the minecraft world border to mark which area gets exported, you just need to do that for the height too. basically a selection box. hope that gave u some ideas.

**Reply by ClockTheSenior** [reply]
oh and mineways has a feature where the world border can be turned off to reduce lag. maybe add that too?

**Reply by Igrium** [reply]
> honestly, 3d would be simpler to implement since you could add a thing like the minecraft world border to mark which area gets exported, you just need to do that for the height too. basically a selection box. hope that gave u some ideas.

Yeah, it will obviously use a selection box internally. I'm more worried about the UI design at the moment.

**Reply by ClockTheSenior** [reply]
Look into mineimator's world exporting feauture since it basically has what u are looking for.
